routes/truckRoutes.js

```diff
@@ -1,2 +1,83 @@
-// Connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
-// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
\ No newline at end of file
+/*eslint-env node*/
+
+//express
+var express = require('express');
+
+//router
+var router = express.Router();
+
+//the food trucks
+var foodTruck = require('../models/truckModel');
+
+router.route('/')
+.get(function (request, response) {
+
+    //return all the trucks
+  foodTruck.find().exec(function (err, result) {
+
+    if (err) {
+
+            //sorry error getting trucks
+        response.status(500).send(err);
+      } else {
+
+            //yay saved
+        response.send(result);
+      }
+
+  });
+})
+
+.post(function (request, response) {
+
+    //create the new truck
+  var newFoodTruck = new foodTruck(request.body);
+
+    //save it
+  newFoodTruck.save(function (err) {
+    if (err) {
+
+            //sorry error saving
+        response.status(500).send(err);
+      } else {
+
+            //yay saved
+        response.status.send(newFoodTruck);
```

Comment:
You're trying to chain off `status`, which is a function, when what you want to do is chain off it's return value, meaning you need to call it.